tokenizers/src/tokenizer/mod.rs

```diff
@@ -643,6 +643,11 @@ where
         final_vocab
     }
 
+    /// Get the added tokens
+    pub fn get_added_tokens(&self) -> Vec<String> {
+        self.added_vocabulary.get_added_tokens()
+    }
```

Comment:
Let's remove this. Vec allocs is costly, you don't want to do this.

Reply:
yep